dev/com.ibm.ws.microprofile.config.1.3/src/com/ibm/ws/microprofile/config13/sources/OSGiConfigUtils.java

```diff
@@ -139,15 +142,22 @@ public static ConfigVariables getConfigVariables(BundleContext bundleContext) {
      * @param bundleContext The context to use to find the service
      * @param serviceClass  The class of the required service
      * @return the service instance or null
+     * @throws InvalidFrameworkStateException
      */
-    public static <T> T getService(BundleContext bundleContext, Class<T> serviceClass) {
+    private static <T> T getService(BundleContext bundleContext, Class<T> serviceClass) throws InvalidFrameworkStateException {
```

Comment:
It looks like this method can no longer return `null` ?

If that's the case, I'd suggest the following changes:
1. Reorder the method so that it's more obvious that it can't return a `null` value
2. Update the `@return` javadoc to say that it can't return `null` (both on this method and on methods which just call this one with a specific `serviceClass`)

In addition, please add javadoc to say when an `InvalidFrameworkStateException` can be thrown. I'd also consider adding to the doc that it can throw a `ServiceNotFoundException` so that any future callers know what will happen if the service can't be found.

Reply:
done

Reply:
The javadoc also needs updating on `getConfigVariables`, `getCDIService` and `getConfigurationAdmin`

Reply:
I guess you decided not to reorder the method?

Reply:
done